tests/helpers/index.ts

```diff
@@ -16,6 +20,17 @@ export function setupProject() {
   });
 }
 
+function addAppToProject(): Promise<any> {
```

Comment:
Not required, but would be nice to have:
Add parameter(s) here to specify values for the app being created.

Reply:
My thinking is: since this function is called by a single caller, it would make it unnecessary complex. 

But I can make a change, if you think it is better this way.